[PROOFREADING] NODE and BUSINESS tutorial section - Spanish

Salgraphics opened the pull request:
Spanish proofread for Bitcoin Knots tutorial. I updated the info about node implementations percentage using the info published in https://coin.dance/nodes 

Screenshots and images are both in English and French, it would be nice to show them in Spanish too.

Salgraphics committed:
proofread es.md

Salgraphics committed:
Update es.md

Proofread Ver. 1  Information about node implementations statistics were taken from: https://coin.dance/nodes

Salgraphics committed:
proofread es.md

There is a paragraph that has not sense, check line 235 at /en.md it reads:  "I recommend that you don't open channels that are too small, bearing in mind that this is a Lightning node and the machine hosting your keys doesn't offer the same level of security as a hardware wallet. So be careful with the amounts you choose to block." I guess it may say Channels that are too big in amount  ?

Salgraphics committed:
proofread es.md

A couple of details... Bitcoin Core's latest version is now 2.9.0 so may be is a good idea to update the info. It looks like part of the info was taken from "Agora256" and its name is still on the tutorial. There is no the "leave us a green thumb up" final sentence but a "feel free to share them with the Agora256 community."

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

There are no links in the tutorial, Text calls for a link on line 22, 49 and 70 but there is no such URL.

MariJJhodl commented:
Hey @Salgraphics thank you for doing this task. 
So, what I get is that you did proofreading for the entire section, and you only modified the following information regarding Bitcoin Knots, right?

- Bitcoin Core es la abrumadora mayoría, con casi el 88% de los nodos de la red Bitcoin utilizando este cliente.
- De ellos, Bitcoin Knots es el más conocido. Actualmente representa alrededor del 11,7% de la red.

Can you confirm?

Salgraphics committed:
proofread 1.1 es.md

Fixed Bold headings

Salgraphics committed:
proofread 1.1 es.md

Fixed bold headings

Salgraphics committed:
proofread 1.1 es.md

Fixed bold headings

Salgraphics committed:
proofread 1.1 es.md

Fixed bold words

Salgraphics committed:
proofread 1.1 es.md

Fixed bold words

Salgraphics committed:
proofread 1.1 es.md

Fixed bold words

Salgraphics committed:
proofread 1.1 es.md

Fixed bold words

Salgraphics committed:
proofread 1.1 es.md

Fixed bold words

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

Thera are some links missing. In line 221 calls for a Wiki but there is no link. On line 456 there are 3 references offering a link but none of them are programmed.

Salgraphics committed:
proofread es.md

Salgraphics committed:
proofread es.md

Salgraphics committed:
Merge branch 'PlanB-Network:dev' into dev

MariJJhodl committed:
Update es.md

MariJJhodl committed:
fix colon

MariJJhodl committed:
fix images lines

MariJJhodl committed:
fix colon

MariJJhodl committed:
fix assets lines 2

MariJJhodl committed:
fix colon

MariJJhodl committed:
fix quotes

MariJJhodl committed:
fix format

MariJJhodl committed:
format fix

MariJJhodl committed:
format fix

MariJJhodl committed:
fix colon

MariJJhodl committed:
fix format

MariJJhodl committed:
fix format

MariJJhodl committed:
format fix

MariJJhodl committed:
fix colon
